primitives/float/addFN.futil

```diff
@@ -0,0 +1,16 @@
+extern "addFN.sv" {
+  primitive addFN[
```

Comment:
Is this implemented combinationally? If so, we should mark it as a `comb primitive` instead of `primitive`. Also, we should probably not implement this as a combinational primitive at all since that is unlikely to meet timing?